<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hexes</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionSave_all"/>
    <addaction name="actionClose_all"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRule">
    <property name="title">
     <string>Rule</string>
    </property>
    <addaction name="actionAdd_Rule"/>
    <addaction name="actionEdit_Rule"/>
    <addaction name="actionPop_Rule"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_Rule"/>
    <addaction name="actionClear_Rules"/>
    <addaction name="separator"/>
    <addaction name="actionRun_Rules"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRule"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Rule">
   <property name="text">
    <string>Add Rule</string>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionEdit_Rule">
   <property name="text">
    <string>Edit Rule</string>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionDelete_Rule">
   <property name="text">
    <string>Delete Rule</string>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionPop_Rule">
   <property name="text">
    <string>Pop Rule</string>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionRun_Rules">
   <property name="text">
    <string>Run Rules</string>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSave_all">
   <property name="text">
    <string>Save all</string>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="text">
    <string>Close all</string>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionClear_Rules">
   <property name="text">
    <string>Clear Rules</string>
   </property>
   <property name="font">
    <font>
     <family>CommitMono</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
